src/db.cpp

```diff
@@ -5542,22 +5542,180 @@ void price_cyber(struct obj_data *obj)
       GET_OBJ_AVAILDAY(obj) = 2;
       break;
     case CYB_SKULL:
-      GET_OBJ_VAL(obj, 1) = 0;
-      GET_CYBERWARE_ESSENCE_COST(obj) = 75;
-      if (GET_OBJ_VAL(obj, 3))
+      GET_OBJ_AVAILDAY(obj) = GET_OBJ_AVAILTN(obj) = GET_CYBERWARE_ESSENCE_COST(obj) = GET_OBJ_COST(obj) = 0;
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_OBVIOUS)) {
```

Comment:
This needs to match the awake.h name, so `SKULL_MOD_OBVIOUS`. Same for other items here.

Reply:
Matched em all up (I think)